rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -714,6 +726,62 @@ int __pal_init_v1(pal_attr_v1_t *attr)
 
 	parse_args(attr->args);
 
+	if (ra_tls_server) {
```

Comment:
In the formal implementation, the logic of tls server should be acted as an application. In other word, `[ "tls", "--server" ]" should appear in args in config.json.